composer.json

```diff
@@ -19,7 +19,7 @@
 	],
 	"require": {
 		"php": ">=5.6",
-		"cakephp/cakephp": "^3.6"
+		"cakephp/cakephp": "^3.7"
```

Comment:
If we need this, we also need to adjust the readme.

Reply:
`Plugin::isLoaded(string)` is a 3.7.0 only feature I believe. I can't really find when `isLoaded` was introduced

Reply:
Or maybe we can submit a backport of `Plugin::isLoaded(string)` to cakephp/cakephp.

Reply:
Would you be opposed as to me implementing a method along `isLoaded()` but in the package, that would do a check based on the CAKE_VERSION and call the appropriate method based on the version?

Reply:
i am just surprised to see tests green after updating the template and before raising the min version.

Reply:
I looked at the build and it appears composer downgraded to 3.7.0 when --prefer-lowest was provided https://travis-ci.org/dereuromark/cakephp-queue/jobs/470218616